src/components/Stories/index.js

```diff
@@ -1,102 +1,113 @@
-import { useMediaQuery } from "@material-ui/core";
-import { useTheme } from "@material-ui/core/styles";
-import { chunk } from "lodash";
+import { CircularProgress } from "@material-ui/core";
 import PropTypes from "prop-types";
 import React, { useEffect, useState } from "react";
 import useSWR from "swr";
 
-import CarouselItem from "./CarouselItem";
+import List from "./List";
 import useStyles from "./useStyles";
 
-import Carousel from "@/pesayetu/components/Carousel";
 import FeaturedStoryCard from "@/pesayetu/components/FeaturedStoryCard";
+import Pagination from "@/pesayetu/components/Pagination";
 import fetchAPI from "@/pesayetu/utils/fetchApi";
-import formatStoryPosts from "@/pesayetu/utils/formatStoryPosts";
-
-const responsive = {
-  desktop: {
-    items: 1,
-  },
-  tablet: {
-    items: 1,
-  },
-};
 
 function Stories({
   featuredStoryProps,
-  items,
+  items: itemsProp,
   category,
   pagination,
+  page,
+  paginate,
   ...props
 }) {
   const classes = useStyles(props);
-  const theme = useTheme();
-  const isTablet = useMediaQuery(theme.breakpoints.up("md"));
-  const itemsToShow = isTablet ? 6 : 3;
   const variant = category === "insights" ? "embed" : undefined;
 
-  // Track all news, including initial news and additionally loaded pages.
-  const [allStories, setAllStories] = useState(items);
-  // Track carousel slide
-  const [nextItemIndex, setNextItemIndex] = useState();
-  const [paginator, setPaginator] = useState(pagination);
-
-  useEffect(() => {
-    setAllStories(items);
-  }, [items]);
+  const [posts, setPosts] = useState(itemsProp);
+  const [isLoading, setIsLoading] = useState(false);
 
-  const fetchMore =
-    paginator?.hasNextPage &&
-    allStories.length - nextItemIndex * itemsToShow <= itemsToShow;
+  const handleClickPage = (_, value) => {
+    if (paginate) {
+      paginate(value);
+    }
+  };
 
-  const { data: moreStoriesProps } = useSWR(
-    fetchMore ? ["/api/wp/archive", category, paginator?.endCursor] : null,
-    (url, taxonomyId, cursor) =>
-      fetchAPI(`${url}/?taxonomyId=${taxonomyId}&cursor=${cursor}`)
+  const { data, error } = useSWR(
+    ["/api/wp/archive", category, page],
+    (url, taxonomyId, pageProp) => {
```

Comment:
What's the advantage of calling useSWR this way instead of say:
```jsx
  const { data, error } = useSWR("/api/wp/archive", (url) => ...rest of fetcher
```

I believe fetcher will still have access to `taxonomyId` and `pageProp`.